src/repl/repl_chat.cpp

```
@@ -278,14 +278,46 @@ static constexpr const char* reminder_nudge =
-  std::string trimmed = line;
+  std::string input = line;  // mutable copy for potential rewrite
+  std::string trimmed = input;
   while (!trimmed.empty() && (trimmed.front() == ' ' || trimmed.front() == '\t')) {
     trimmed.erase(trimmed.begin());
   }
   if (trimmed.empty()) {
     s.out << "[empty prompt — type something or /help]\n";
     return;
   }
+  // Intercept "y"/"yes"/"ja" after model suggested a command in plain text
+  if ((trimmed == "y" || trimmed == "yes" || trimmed == "ja") && !s.history.empty()) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)